src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue

```diff
@@ -315,61 +308,46 @@ export default {
 			}
 		},
 
-		getVotePercentage() {
-			return (index) => {
-				if (this.pollVotes[`option-${index}`] === undefined) {
-					return 0
-				}
-				return parseInt(this.pollVotes[`option-${index}`] / this.votersNumber * 100)
-			}
-		},
-
-		/**
-		 * Current actor id
-		 */
-		actorId() {
-			return this.$store.getters.getActorId()
+		participantType() {
+			return this.$store.getters.conversation(this.token).participantType
 		},
 
-		/**
-		 * Current actor type
-		 */
-		actorType() {
-			return this.$store.getters.getActorType()
+		selfIsOwnerOrModerator() {
+			return (this.poll.actorType === this.$store.getters.getActorType() && this.poll.actorId === this.$store.getters.getActorId())
```

Comment:
Now I feel, we need an util function for this :D

Reply:
Maybe a getter? I tried to implement it, as `selfIsActor`, and it shows itself pretty good. Though we have a different checks in different places for this, so should be unified first